sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -62,7 +62,11 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
 ].forEach((testCase) => {
   describe(`Atom management - Basic CRUD on "${testCase.entityType}" entities`, function(): void {
     before(async () => {
-      await createEntity(EntityType.TOPIC, alwaysBeExistingTopic);
+      try {
+        await createEntity(EntityType.TOPIC, alwaysBeExistingTopic);
+      } catch (err) {
+        // Create and ignore if already created
```

Comment:
Can the exception occur for other reasons than the fact that it was already created?

Reply:
Yes, such as network failures.
This handling can in general be improved once we use unique test resources for each tests.